### ISSUE TITLE
Table Formatting problem while using Jspdf.Table function 

### ISSUE THREAD

**Issue by girish0628** [yWorks/jsPDF#3]
Hi,
I am using the latest fork version of jspdf (**Version 1.2.68** ) to print a html table , but the resulting table has some serious issues. The resulted table header is getting merged with the row texts and the fill color of table header is same as that of text. I tried modifying this fork  api by comparing it with Original MrRio version but was not able to figure it out correctly.  Any help in this regards will be highly appreciated.

P.S: I am using this fork version as I have svg charts which needs to be printed

Thanks
Girish

**Comment by yGuy** [comment]
I think this is actually a duplicate of #2

**Comment by yGuy** [comment]
This will hopefaully start working again once #13 is fixed. Basically this issue is blocked by #13

**Comment by HackbrettXXX** [comment]
See my latest comment on #2 ;)
